src/Xamarin.Android.Build.Tasks/Tasks/Aapt2LinkAssetPack.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using Xamarin.Android.Tools;
+using Microsoft.Android.Build.Tasks;
+
+namespace Xamarin.Android.Tasks {
+
+	public class Aapt2LinkAssetPack : Aapt2 {
+		public override string TaskPrefix => "A2LAP";
+
+		[Required]
+		public ITaskItem Manifest { get; set; }
+
+		[Required]
+		public ITaskItem[] AssetDirectories { get; set; }
+
+		[Required]
+		public string PackageName { get; set; }
+
+		[Required]
+		public ITaskItem OutputArchive { get; set; }
+
+		public string OutputFormat { get; set; } = "proto";
```

Comment:
This feels a tad odd: a public property, used below, but the callsite within `Xamarin.Android.Assets.targets` doesn't specify this property.

Why does it exist?  The only way it could be set is from unit tests, or someone explicitly `<UsingTask/>` the type and invoking it themselves.